StreamChatUITestsAppUITests/MockServer/ChannelResponses.swift

```diff
@@ -57,15 +74,83 @@ extension StreamMockServer {
             return (channel?[ChannelCodingKeys.id.rawValue] as? String) == id
         }) {
             let messageList = findMessagesByChannelId(id)
-            channels?[index][ChannelQuery.CodingKeys.messages.rawValue] = messageList
+            channels?[index][ChannelPayload.CodingKeys.messages.rawValue] = messageList
             json[TopLevelKey.channels] = channels
             currentChannelId = id
             channelList = json
         }
         
         return .ok(.json(json))
     }
-    
+
+    private func addMembersToChannel(_ request: HttpRequest, ids: [String]) -> HttpResponse {
+        var json = channelList
+        guard let id = request.params[EndpointQuery.channelId] else { return .ok(.json(json)) }
+
+        var channels = json[TopLevelKey.channels] as? [[String: Any]]
+        if let index = channels?.firstIndex(where: {
+            let channel = $0[ChannelPayload.CodingKeys.channel.rawValue] as? [String: Any]
+            return (channel?[ChannelCodingKeys.id.rawValue] as? String) == id
+        }) {
+            guard var channel = channels?[index],
+                  var members = channel[ChannelCodingKeys.members.rawValue] as? [[String: Any]] else {
+                return .badRequest(nil)
+            }
+
+            let addedMembers = memberJSONs(for: ids)
+            members.append(contentsOf: addedMembers)
+            channel[ChannelCodingKeys.members.rawValue] = members
+            channel[ChannelCodingKeys.memberCount.rawValue] = members.count
+            channels?[index] = channel
+            json[TopLevelKey.channels] = channels
+
+            channelList = json
+
+            if let channelId = (channel["channel"] as? [String : Any])?["id"] as? String {
+                addedMembers.forEach {
+                    websocketMember(with: $0, channelId: channelId, eventType: .memberAdded)
+                }
+                websocketChannelUpdated(with: members, channelId: channelId)
+            }
+        }
+
+        return .ok(.json(json))
+    }
+
+    private func removeMembersFromChannel(_ request: HttpRequest, ids: [String]) -> HttpResponse {
```

Comment:
`addMembersToChannel` and `removeMembersFromChannel` methods look pretty similar at the first glance. Any chance we could pull something out to a new common method?

Reply:
done